### PR TITLE
📝 adding content to about page

### PR DESCRIPTION
**Pull request by Ricardo24Heredia**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -547,6 +604,22 @@
       }
     }
   }
+
+  /* WE ARE SECTION */
+  #we_are-section {
+    min-height: 100vh;
+
+    &::before {
```
Comment (Doisaac) [inline css/style.css:612]: We should delete this

Comment (Doisaac) [inline css/style.css:612]: We could delete this pseudo-element

Comment (Doisaac) [inline css/style.css:612]: We could delete this pseudo-element

**Review by Doisaac — CHANGES_REQUESTED** [review]

File: css/style.css [hunk 2]
```diff
@@ -547,6 +604,22 @@
...
+      background-size: 100vw calc(100% + 400px);
+      background-position: 0 -314px;
+    }
```
Comment (Doisaac) [inline css/style.css:615]: We could delete this unused pseudo-element

File: css/style.css [hunk 3]
```diff
@@ -547,6 +604,22 @@
       }
     }
   }
+
+  /* WE ARE SECTION */
+  #we_are-section {
+    min-height: 100vh;
+
+    &::before {
+      background-size: 100vw calc(100% + 400px);
```
Comment (Doisaac) [inline css/style.css:613]: We could delete this unused pseudo-element

File: css/style.css (hunk 1 above)
Comment (Doisaac) [inline css/style.css:612]: We could delete this unused pseudo-element

File: css/style.css (hunk 2 above)
Comment (Doisaac) [inline css/style.css]: We could delete this unused pseudo-element

**Review by Doisaac — CHANGES_REQUESTED** [review]

**Review by Doisaac — APPROVED** [review]

**Review by RNavarro23 — APPROVED** [review]

**Pull request merged**